OctoAwesome/OctoAwesome.Client/Components/ChunkRenderer.cs

```diff
@@ -210,8 +208,7 @@ public bool RegenerateVertexBuffer()
                         if (blockDefinition == null)
                             continue;
 
-                        int textureIndex;
-                        if (!textureOffsets.TryGetValue(blockDefinition, out textureIndex))
+                        if (!textureOffsets.TryGetValue(blockDefinition, out int textureIndex))
                             continue;
```

Comment:
Verstößt gegen die Coding-Richtlinien

out-Variablen aus if-Statements / "is" in if

Reply:
leute leute bitte das glaub ich noch vom guten bobstriker persönlich glaub ich :)

Reply:
lasst den Code einfach erst mal ruhen am besten, der muss eh noch stark angepasst werden und ist uralt, die meisten Änderungen waren Formatierungsänderungen (oder paar Kleinigkeiten von mir).
Aber wie gesagt, den braucht man mMn nicht reviewen, denn auch wie der vorher war wär er wahrscheinlich nicht gültig und muss eh noch komplett geändert werden